lib/workers/repository/process/lookup/index.ts

```diff
@@ -50,6 +51,8 @@ export interface LookupUpdateConfig
   vulnerabilityAlert?: boolean;
   separateMajorMinor?: boolean;
   separateMultipleMajor?: boolean;
+  datasource: string;
+  depName: string;
```

Comment:
Do we need this here?

Reply:
![image](https://user-images.githubusercontent.com/6311784/99252129-cc77f980-280e-11eb-9d39-44ff46aed67e.png)

Reply:
I'm not sure why this check fails due to this PR but I'm ok to go with this unless you have something better